src/main/resources/hudson/plugins/matrix_configuration_parameter/MatrixCombinationsParameterDefinition/index.groovy

```diff
@@ -66,7 +63,7 @@ private void drawMainLinksJS(String nameIt) {
             "var element = document.parameters.elements[i];\n" +
```

Comment:
@ikedam notice the use of `document.{formname}.elements` here which is all that I copy down below